Tasks/GradleV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -47,9 +47,15 @@
   "loc.input.label.checkstyleAnalysisEnabled": "Run Checkstyle",
   "loc.input.help.checkstyleAnalysisEnabled": "Run the Checkstyle tool with the default Sun checks. Results are uploaded as build artifacts.",
   "loc.input.label.findbugsAnalysisEnabled": "Run FindBugs",
-  "loc.input.help.findbugsAnalysisEnabled": "Use the FindBugs static analysis tool to look for bugs in the code. Results are uploaded as build artifacts.",
+  "loc.input.help.findbugsAnalysisEnabled": "Use the FindBugs static analysis tool to look for bugs in the code. Results are uploaded as build artifacts. In Gradle 6.0 this plugin was removed. Use spotbugs plugin instead. [More info](https://docs.gradle.org/current/userguide/upgrading_version_5.html#the_findbugs_plugin_has_been_removed)",
   "loc.input.label.pmdAnalysisEnabled": "Run PMD",
   "loc.input.help.pmdAnalysisEnabled": "Use the PMD Java static analysis tool to look for bugs in the code. Results are uploaded as build artifacts.",
+  "loc.input.label.spotBugsAnalysisEnabled": "Run SpotBugs",
+  "loc.input.help.spotBugsAnalysisEnabled": "Enable this option to run spotBugs. This plugin works with Gradle v5.6 or later. [More info](https://spotbugs.readthedocs.io/en/stable/gradle.html#use-spotbugs-gradle-plugin)",
+  "loc.input.label.spotBugsGradlePluginVersionChoice": "Spotbugs plugin for Gradle version",
```

Comment:
This sounds little bit confusing (plugin or gradle version?)

Reply:
@anatolybolshakov Okay, corrected it.

Reply:
It's still confusing :)

![image](https://user-images.githubusercontent.com/16704239/113412233-9aee6d80-936c-11eb-9992-2f3d9a30094c.png)